tsl/src/bgw_policy/retention_api.c

```diff
@@ -262,7 +269,7 @@ policy_retention_add(PG_FUNCTION_ARGS)
 	namestrcpy(&owner, GetUserNameFromId(owner_id, false));
 
 	job_id = ts_bgw_job_insert_relation(&application_name,
-										&default_schedule_interval,
+										default_schedule_interval,
 										&default_max_runtime,
 										default_max_retries,
 										&default_retry_period,
```

Comment:
Actually none of the `ts_bgw_job_insert_relation` parameters require to be passed by reference or am I missing something?